### DISCUSSION TITLE
STAR presentation on ND not correct compared with Navigraph

### DISCUSSION THREAD

**Discussion by Akkifriman** [flybywiresim/aircraft#2384] [Help]
I have found that some STARs are not provided correctly on the ND for some reason. My example is from the MONA1N arrival to Copenhagen airport Kastrup EKCH from the south expecting rwy 22L.
According to the Navigraph map it should take a north-east route and final after a left turn to intercept with ILS rwy 22L, however the latest mod of A320NX draw on the ND a completely different route, see below:
![Skärmklipp7](https://user-images.githubusercontent.com/70971118/101988031-973cbb00-3c97-11eb-946d-df9f97487b8a.JPG)
![Missing STAR EKCH_LI](https://user-images.githubusercontent.com/70971118/101988058-cd7a3a80-3c97-11eb-9229-64010122c7f8.jpg)
![MONAK1N](https://user-images.githubusercontent.com/70971118/101988068-e1be3780-3c97-11eb-88e0-59a36d39e3aa.JPG)
![MONAK1Nno2](https://user-images.githubusercontent.com/70971118/101988072-eedb2680-3c97-11eb-840d-53d56da86b0c.JPG)

What can be done here? There are STARS that are correct, others not, why? Navigraph beta corrupt?

**Comment by NathanInnes** [comment]
This happens for the majority of arrivals in MSFS, sounds more of an adobo thing to fix.

**Reply by Akkifriman** [reply]
So the programming of the MCDU is not in the scope of FBW guys?

**Reply by Akkifriman** [reply]
If I dont apply the ILS 22L from the ARRIVAL page and put in the ILS frequency manually on the RAD/NAV page I get the correct STAR route:
![Skärmklipp](https://user-images.githubusercontent.com/70971118/102008691-aae92f80-3d32-11eb-85a5-438ebfd3ed9d.JPG)

**Reply by Lucky38i** [reply]
can I see your arrival page on the fplan page?

**Reply by Akkifriman** [reply]
Not sure I understand what it is you want to see? This one?
![Skärmklipp7](https://user-images.githubusercontent.com/70971118/102366236-b3eb3280-3fb8-11eb-9712-eccfbd48bf0c.JPG)

**Reply by Akkifriman** [reply]
If I do this - this is the result:
![MONAK1Nno2](https://user-images.githubusercontent.com/70971118/102366352-daa96900-3fb8-11eb-8bdf-6c54efa4ff87.JPG)
It should be as of above 1st picture

**Reply by Lucky38i** [reply]
what happens if you delete the USR waypoint

**Reply by Akkifriman** [reply]
My experience is that I am note able to delete a wp...but I might be doing something wrong.

**Reply by HiggsBoson777** [reply]
I flew 2 flights today and the STAR was completely off for both. The first was the Habbs5 Arrival into Montreal (CYUL) and the other was the Canuck5 Arrival in Vancouver (CYVR). Also, I did not have an USR waypoint to delete, the waypoints were just completely different from the approach plates.

**Reply by StefNx** [reply]
Actually I started having problems both with SIDs and STARs since the new updates. I have the latest AIRAC pulled from Navigraph on Simbrief but somehow it is missing many SIDs on the A32NX system even for big airports.

**Reply by Akkifriman** [reply]
Its bin a problem since August 18th actually and I was hoping that Asobo would have solved the problem with the 22nc december update but this is not the case. GitHub claim its hardcoded so they cannot do anything but Asobo. Its not a Navigraph beta issue either, data is there but the interpretation on the ND(MCDU) is corrupt.
